src/lib/time-picker/time-picker.ts

```diff
@@ -217,6 +222,9 @@ export class TimePickerComponent extends BaseComponent implements ITimePickerCom
   @FoundationProperty()
   public showNow: boolean;
 
+  @FoundationProperty()
+  public showHourOptions: boolean;
```

Comment:
I know the rest of this file doesn't have one, but might be nice to add some JSDoc to this property with a description.  For now you'd probably want to duplicate it in `time-picker-args.ts`, but the comment here would get picked up by the Custom Elements Manifest and could eventually be used to generate the documentation.